po/zh-Hans.po

```diff
@@ -2002,13 +2002,15 @@ msgstr ""
 
 #: src/hello-world.md:1
 msgid "# Hello World!"
-msgstr ""
+msgstr "# Hello World!"
 
 #: src/hello-world.md:3
 msgid ""
 "Let us jump into the simplest possible Rust program, a classic Hello World\n"
 "program:"
 msgstr ""
+"让我们进入最简单的 Rust 程序，一个经典的 Hello World\n"
+"程序："
```

Comment:
These 2 lines can be merged into one line by removing the "\n" after "Hello World" at line 2009